.travis.yml

```diff
@@ -4,16 +4,10 @@ python:
 - 3.7
 - 3.6
 - 3.5
-before_install: | 
```

Comment:
This will remove the check we had earlier - it will now build for every commit even for non-code related changes, are you fine with that?